content/concept-depositauth.md

```diff
@@ -0,0 +1,59 @@
+- Can receive XRP from [PaymentChannelClaim transactions][] **only if** the sender of the PaymentChannelClaim transaction is the destination of the payment channel.
+- Can receive XRP from [EscrowFinish transactions][] **only if** the sender of the EscrowFinish transaction is the destination of the escrow.
+- **Can** receive XRP or issued currencies by sending a [CheckCash][] transaction. _(Requires the [Checks amendment](reference-amendments.html#checks).)_
+- **Can** receive XRP or issued currencies by sending [OfferCreate transactions][]. ***TODO: test this***
+    - If the account sends an OfferCreate transaction that is not fully executed immediately, it **can** receive the remainder of the ordered XRP or issued currency later when the offer is consumed by other accounts' [Payment][] and [OfferCreate][] transactions. ***TODO: test this***
+- If the account has created any trust lines, or issued currency on trust lines created by other accounts, the account **can** receive the issued currencies of those trust lines in [Payment transactions][] as a result of [rippling](concept-noripple.html). It cannot be the destination of those transactions. ***TODO: test this***
+    - If the account has not created any trust lines with a nonzero limit, and has not issued currency on trust lines created by others, it **cannot** receive any issued currencies in any transaction.
+
+The following table summarizes whether a transaction type can deposit money with DepositAuth enabled or disabled:
+
+{% include 'snippets/depositauth-semantics-table.html' %}
+
+
+## Enabling or Disabling Deposit Authorization
+
+An account can enable deposit authorization by sending an [AccountSet transaction][] with the `SetFlag` field set to the `asfDepositAuth` value (9). The account can disable deposit authorization by sending an [AccountSet transaction][] with the `ClearFlag` field set to the `asfDepositAuth` value (9). For more information on AccountSet flags, see [AccountSet flags](reference-transaction-format.html#accountset-flags).
+
+## Checking Whether an Account Has DepositAuth Enabled
+
+To see whether an account has Deposit Authorization enabled, use the [`account_info` command](reference-rippled.html#account-info) to look up the account. Compare the value of the `Flags` field (in the `result.account_data` object) with the [boolean flags defined for an AccountRoot ledger object](reference-ledger-format.html#accountroot-flags).
+
+If the result of the `Flags` value boolean-AND the `lsfDepositAuth` flag value (0x01000000) is nonzero, then the account has DepositAuth enabled. If the result is zero, then the account has DepositAuth disabled.
```

Comment:
> boolean-AND

Typo?

Reply:
Not a typo. "boolean AND" is the name of [the operation](https://en.wikipedia.org/wiki/Boolean_algebra#Basic_operations). It's the same sentence structure as "X plus Y" but instead of "plus" the operation is "boolean-AND".

I'm open to more ways to phrase this more clearly. I can't think of one at the moment.

Reply:
I think the desired operation is actually the [bitwise AND](https://en.wikipedia.org/wiki/Bitwise_operation#AND)?

Reply:
Yes, you're absolutely right. Apologies to @ryangyoung—I completely overlooked this vital mistake because I was so maniacally focused on it not being a typo. Mea cupla; the problem isn't in the keyboard but in the chair.

Will change to say bitwise-AND.